crates/util/python-utils/src/constants.rs

```diff
@@ -32,6 +27,17 @@ pub(crate) static DESCRIPTOR: LazyLock<&'static str> =
 pub(crate) static CHANGE_DESCRIPTOR: LazyLock<&'static str> =
     LazyLock::new(|| Box::leak(format!("tr({XPRIV}/86'/1'/0'/1/*)").into_boxed_str()));
 
+/// Corresponds to address length of Execution environment.
+pub(crate) const EE_ADDRESS_LEN: u8 = 20;
```

Comment:
We should also define this in the `strata-primitives` `constants` module similar to this: https://github.com/alpenlabs/alpen/blob/ababb1f7d79993bdcb154223dc4240daef953bac/crates/primitives/src/constants.rs#L20-L22